ui/app/templates/vault/cluster/access/identity/index.hbs

```diff
@@ -110,12 +110,12 @@
       this.identityType
     }} in this namespace will be listed here. Create your first {{this.identityType}} to get started."
   >
-    <LinkTo @route="vault.cluster.access.identity.create" @model={{pluralize this.identityType}} class="link">
+    <Hds::Link::Standalone @route="vault.cluster.access.identity.create" @model={{pluralize this.identityType}}>
```

Comment:
I think this will throw an error because `@icon` is required for standalone links

Reply:
yep
```
Uncaught (in promise) Error: Assertion Failed: @icon for "Hds::Link::Standalone" must have a valid value
    at assert (vendor.js:18413:15)
    at get icon [as icon] (vendor.js:123937:59)
    at getPossibleMandatoryProxyValue (vendor.js:10500:19)
    at _getProp (vendor.js:10531:17)
    at vendor.js:38169:45
    at vendor.js:38117:37
    at track (vendor.js:47684:7)
    at valueForRef (vendor.js:38116:44)
    at vendor.js:38208:40
    at vendor.js:38117:37